### ISSUE TITLE
Release?

### ISSUE THREAD

**Issue by alex-ong** [VodBox/filter-pack#2]
Is there a place i can get a compiled version of this? I would love to promote corner pinning as a cheap way of capturing game footage when the end-user doesn't have a capture card.

**Comment by VodBox** [comment]
I do not have a compiled version of this ready to go, no. Part of the issue, is that it's written with the assumption of a source being 16:9, and only has bilinear interpolation, as opposed to perspective, which would look nicer.

I'm uncertain what you mean though. Cheap way of capturing without a capture card? Do you mean applying to a webcam source pointed at a screen?

**Comment by alex-ong** [comment]
Yes - webcam source pointed at a screen then corner pinned back to rectangle. There are a few perspective plugins available but they all are calibrated as XYZ rotation rather than 4 corners, which is more cumbersome.

I've made a branch that does bilinear interpolation from four points back to rectangle, however the closer edge is larger than the far edge. 

By using a proper perspective transform, the correct viewing angle can be recreated. Unfortunately all the guides I read with the vector math went over my head.
